docs/BUILD.md

```diff
@@ -0,0 +1,41 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents** 
+
+- [Release workflow](#release-workflow)
+  - [Step 1: Release](#step-1-release)
+  - [Step 2: Publish](#step-2-publish)
+  - [Checklist](#checklist)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Release workflow
+
+The main goal of the build process is to build and publish the docker image to the docker hub repo: 'commercetools/commercetools-project-sync'
+     
+## Step 1: Release
+
+Create a PR for the new release: 
+- Increment the release version to the new library version, please follow the [semantic versioning](https://semver.org/) for finding the new version.
+- Make sure all the dependencies are up-to-date everywhere in the documentation files and project files.
+- Make sure to update the [release notes](/docs/RELEASE_NOTES.md). 
+- Ask for review for this PR and then "squash and merge" to master.
+
+
+------
+    
+To release the library, you need to ["create a new release"](https://github.com/commercetools/commercetools-project-sync/releases/new) with Github, 
+describe the new release and publish it. 
+
+For example, The release description should show of new features, bug fixes and updates of the dependencies versions(https://github.com/commercetools/commercetools-project-sync/releases/tag/3.10.0)
```

Comment:
Why not using a markdown link ? instead of text link ?

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/231/commits/b73ecf1ffb66a26b29c21d81cfea3d4b8ce10440